Resources/doc/index.md

```diff
@@ -1,12 +1,25 @@
-Get starting
-============
-
-1. [Required dependencies](index.md#required-dependencies)
-2. [Installation](index.md#installation)
-3. [Usage](usage.md)
-4. [Configuration](config.md)
-5. [Event](events.md)
-6. [FAQs](faqs.md)
+Getting started
+===============
+
+1. [Introduction](index.md#introduction)
+2. [Required dependencies](index.md#required-dependencies)
+3. [Installation](index.md#installation)
+4. [Usage](usage.md)
+5. [Configuration](config.md)
+6. [Event](events.md)
+7. [FAQs](faqs.md)
+
+## Introduction
+
+Foxy is a Composer plug-in that aggregates npm-packages from Composer packages.
+
+This makes it possible (and automates the process of) installing and updating npm-packages that ship with your Composer packages, leveraging the native (`npm` or `yarn`) package manager to do the heavy lifting.
+
+For this approach to work well, you should think of an npm-package in a Composer package not just as an "artifact", but as an actual npm-package *embedded* in your Composer package.
```

Comment:
I don't really agree to this paragraph, since to my understanding the `npm` packages are less tied to composer packages with foxy.

Reply:
"less tied", maybe, but they're still directly dependent on updating the Composer package? the only way to update the npm-package is by updating the Composer package, since they're literally in the same source-repo.

feel free to change/remove anything you want though, it's your project - just trying to help :-)

Reply:
> the only way to update the npm-package is by updating the Composer package

You mean the "npm-package constraints" ? Maybe it's just a misunderstanding or terminology... 

> it's your project - just trying to help :-)

Me too :)

Reply:
maybe my thinking/understanding is different from yours?

> For this approach to work well, you should think of an npm-package in a Composer package not just as an "artifact"...

By this I mean, a lot of people currently create an npm-package with no name or version number - an "artifact" in the sense that this isn't really a package that ever gets installed, it's just a means of setting up your Composer project's client-side requirements.

> ...you should think of an npm-package in a Composer package ... as an actual npm-package *embedded* in your Composer package.

That is, I don't think anybody is going to push a built-in npm-package to a repository server - they're most likely just looking to use this tool as a means of managing client-side dependencies built-into a Composer package.

Maybe my understanding is completely different from yours :-)

Reply:
as said, feel free to rephrase or change anything I wrote - I'm just trying to formulate it in a way where I think I'd have been able to understand it myself :-)